data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py

```diff
@@ -69,40 +73,18 @@ def _create_string_from_list(series: pd.Series) -> str:
         return ", ".join(str_list)
 
     @staticmethod
-    def _adjust_percentage_to_string(percentage_float: float) -> str:
-        """Helper method that converts numeric floats to strings based on what-to-show rules.
-
-        What are these rules?
-        0. If None, return none
-        1. If the percentage is below 1%, produce 'less than 1%'
-        2. If the percentage is above 99.95%, produce '100%'
-        3. If the percentage is X.00 when rounded to two sig digits, display the integer of the percent
-        4. If the percentage has unique significant digits, report two digits
-        """
-        # Rule 0
-        if not percentage_float:
-            # I believe we need to do this because JS will do weird things with a mix-type column?
-            return "No tribal areas"
-        # Rule 1
+    def _adjust_percentage_for_frontend(
+        percentage_float: float,
+    ) -> Optional[float]:
+        """Round numbers very close to 0 to 0 and very close to 1 to 1 for display"""
+        if percentage_float is None:
+            return None
         if percentage_float < 0.01:
-            return "less than 1%"
-        # Rule 2
+            return 0.0
         if percentage_float > 0.9995:
-            return "100%"
-
-        rounded_percentage_str = str(round(percentage_float, 4) * 100)
-        first_digits, last_digits = rounded_percentage_str.split(".")
-
-        # Rule 3 (this is a shorthand because round(4) will truncate repeated 0s)
-        if last_digits[-1] == "0":
-            return first_digits + "%"
+            return 1.0
 
-        # Rule 4
-        if last_digits != "00":
```

Comment:
We're dropping the whole rule of "Convert `7.00%` to `7`"?

Reply:
We're returning floats now, so it'll be on the frontend to do that